manage.py

```diff
@@ -232,51 +251,75 @@ def pre_commit():
-                    if schema_reference[j][:10] == ":collapse:":
-                        components[component].append(f":collapse: {','.join(schema['definitions'][component]['properties'].keys())}\n")
-                    else:
-                        components[component].append(schema_reference[j])
-                    j += 1
 
-    # Add components for new definitions
-    for key, value in schema["definitions"].items():
-        if key not in components:
-            components[key] = [
-                f"A `{key}` is defined as:\n",
-                "```{jsoninclude-quote} ../../schema/network-schema.json\n",
-                f":jsonpointer: /definitions/{key}/description\n",
-                "```\n",
-                f"Each `{key}` has the following fields:\n", 
-                "```{jsonschema} ../../schema/network-schema.json\n",
-                f":pointer: /definitions/{key}\n",
-                f":collapse: {','.join(value['properties'].keys())}\n"
-                "```\n",
-                "\n"
-            ]
+                # Drop auto-generated reference content
+                while j < len(schema_reference) and not schema_reference[j].startswith("```{admonition}") and not schema_reference[j].startswith(f"`{defn}"):
+                    schema["definitions"][defn]["content"].append(schema_reference[j])
+                    j = j+1
+
+    # Generate standard reference content for each definition
+    for defn, definition in schema["definitions"].items():
+        definition["content"] = definition.get("content", [])
         
+        # Add heading
+        definition["content"].insert(0, f"#### {defn}\n")
+        
+        # Get Github issues and list related definitions and properties
+        definition["issues"] = {}
+        if definition.get("$comment"):
+            for issue in get_issues(definition["$comment"]):
+                definition["issues"][issue.url] = {"issue": issue, "relatedTo": [defn]}
+        
+        for prop, property in definition["properties"].items():
+            if property.get("$comment"):
+                for issue in get_issues(property["$comment"]):
+                    if issue.url in definition["issues"]:
+                        definition["issues"][issue.url]["relatedTo"].append(f".{prop}")
+                    else:
+                        definition["issues"][issue.url] = {"issue": issue, "relatedTo": [f".{prop}"]}
+          
+        # Add admonition with list of related Github issues
+        if len(definition["issues"]) > 0:
+            definition["content"].extend([
+                "```{admonition} Alpha consultation\n",
+                "The following issues relate to this component or its fields:\n"
+            ])
+            for issue in definition["issues"].values():
+                definition["content"].extend([
+                    f"* `{'`, `'.join(issue['relatedTo'])}`: [#{issue['issue'].number} {issue['issue'].title}]({issue['issue'].url})\n"
```

Comment:
```suggestion
                    f"* `{'`, `'.join(issue['relatedTo'])}`: [#{issue['issue'].number} {issue['issue'].title}]({issue['issue'].html_url})\n"
```

Unless I'm confused, we want to link to the issue url rather than the api url, right?

Reply:
Good spot :-)